checkov/terraform/checks/graph_checks/aws/PostgresDBHasQueryLoggingEnabled.yaml

```diff
@@ -0,0 +1,37 @@
+metadata:
+  name: "Postgres RDS as aws_db_instance has Query Logging enabled for "
+  id: "CKV2_AWS_30"
+  category: "LOGGING"
+definition:
+  or :
+    - cond_type: filter
+      attribute: resource_type
+      value:
+      - aws_db_instance
+      operator: within
+    - cond_type: filter
+      resource_types:
+      - aws_db_instance
+      attribute: engine
+      operator: within
+      value: "postgres"
```

Comment:
@JamesWoolfenden Using the `equals` operator is a little faster than the `within` operator

Reply:
I dont think the filter supports equals only within:
operator_to_filter_solver_classes = {
    "within": WithinFilterSolver,
}

Reply:
I meant replaing the entire block
```yaml
- cond_type: attribute
  attribute: engine
  operator: equals
  value: postgres
  resource_types:
    - aws_db_instance
```

Reply:
that keeps the resource as passed and i want them excluded